README.md

```diff
@@ -1 +1,3 @@
-# Catch_it
\ No newline at end of file
+# Catch_it
+## Authors 
+[ Gajera Chintan} https://github.com/gajerachitan9
```

Comment:
It should end with box bracket [Header](link)

Reply:
done thanks can you please approve